grant-project/grant-monorepo
# Header, Footer & Home styled to less conversion

Part of #41 

### What we have here
A conversion of the Header, Home and Footer components' styles from styled-component to less.

### Test
- Visit / make sure looks correct at wide and narrow widths, button hover effects intact.
- Click through to /create or /proposals and observe that the header and footer render correctly.
